client/packages/common/src/ui/components/steppers/VerticalStepper/VerticalStepper.tsx

```diff
@@ -49,7 +49,7 @@ const StyledConnector = styled(StepConnector)(({ theme }) => ({
 }));
 
 const Circle = (props: StepIconProps) => {
-  const { active, completed, ...rest } = props;
+  const { active, completed, error, ...rest } = props;
```

Comment:
Why add this? Just getting a not used error now..

Reply:
yup. it's spreading props which applies `error` as a prop on a `<div/>` and gives a grumpy:

<img width="889" alt="image" src="https://github.com/user-attachments/assets/108bf2a9-b57c-4ce9-8a00-44df3fc57570">

Not sure why the unused var complaint didn't show 🤔 Will fix it up 🙏